### DISCUSSION TITLE
Angular PrimeNG Tree Select Filter Issue

### DISCUSSION THREAD

**Discussion by Parthiban1593** [primefaces/community#2599] [PrimeNG]
I faced a issue in treeselect component when i used to render treeselect component under mat-dialog-content directive with filter=true the tree dropdown panel was not open. Either i need to remove the ma-dialog-content directive or remove filter to display dropdown panel. Im using primeng 13 version.

**Comment by qwadrox** [comment]
Angular Material's dialogs have their own overlay mechanism which might interfere with the positioning and rendering of PrimeNG's panel.
The dropdown panel might be opening but is hidden behind the dialog due to z-index or other CSS issues. 

Did you try to use [appendTo](https://primeng.org/treeselect#api.treeselect.props.appendTo)="body"?

The interaction between Angular Material and PrimeNG components can sometimes cause conflicts. it's generally better to use a single UI library in a project rather than mixing multiple libraries.

In this case you could use PrimeNg Dialog instead of mat dialog.
